### PR TITLE
adding JS snippet

### PR DESCRIPTION
**Pull request by sannesofie**
### General Checks

- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work
- [ ] any embedded code snippets are formatted

### CODE REVIEW

File: snippets/javascript/statements.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+```js
+const isMailSent = true;
+
+if (isMailSent) {
+  console.log("Mail sent to recipient");
+}
+```
+
+<!-->An if statement accepts an expression with a set of parentheses:
```
Comment (arnochauveau) [inline snippets/javascript/statements.md:13]: I think the `<!-->` in your MD file are a mistake, or are you trying to achieve something with them? 😄  There are several of them, so please remove/change them all.

File: snippets/javascript/statements.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const isTaskCompleted = false;
+
+if (isTaskCompleted) {
+  console.log("Task completed");
+} else {
+  console.log("Task incomplete");
+}
+```
+
+<!-->An else block can be added to an if block or series of if-else if blocks. The else block will be executed only if the if condition fails.<!-->
```
Comment (arnochauveau) [inline snippets/javascript/statements.md:31]: Fails is not the right word here. Failing means something else in programming (throwing an error). I would reword to `if the if condition is falsy`

File: snippets/javascript/statements.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+if (size > 100) {
+  console.log("Big");
+} else if (size > 20) {
+  console.log("Medium");
+} else if (size > 4) {
+  console.log("Small");
+} else {
+  console.log("Tiny");
+}
+// Print: Small//
```
Comment (arnochauveau) [inline snippets/javascript/statements.md:47]: copied by accident?

**Review by arnochauveau — CHANGES_REQUESTED** [review]
Very good, Sanne! You seem to have a good understanding of the if-, if else- and else statements. 

There are a few weird things that got in to your MD file that you should change. 
There is also a small suggestion for your else part in the comments.

Keep up the good work!

File: snippets/javascript/statements.md (hunk 3 above)
Comment (sannesofie) [reply snippets/javascript/statements.md:47]: Hi @arnochauveau Not sure what you mean by copied by accident?

Comment (arnochauveau) [reply snippets/javascript/statements.md:47]: The `// Print: Small //` in your code? 😄 I think it was copied by accident.

Comment (sannesofie) [reply snippets/javascript/statements.md:47]: It means that the this code will print Small to the console I thought, maybe that is wrong

Comment (arnochauveau) [reply snippets/javascript/statements.md:47]: It's a comment, so it would surprise me if it did. 

You can style the log result with css (example font-size) by using `console.log('%c Your text', 'font-size: 10px; other-css-properties: value')`. This works for chrome, not sure about other browsers.

Anyway, the snippets should be focused on the topic of the snippet as all the extra stuff might be confusing for others 🙂

Comment (sannesofie) [reply snippets/javascript/statements.md:47]: Ok no worries I will remove it

**Review by arnochauveau — APPROVED** [review]
